src/design/components/atoms/LeftRightList.tsx

```diff
@@ -1,13 +1,16 @@
 import React, { useRef } from 'react'
 import { useEffectOnce } from 'react-use'
+import { focusFirstChildFromElement } from '../../../cloud/lib/dom'
 import { useLeftToRightNavigationListener } from '../../lib/keyboard'
 
 interface LeftToRightListProps {
   className?: string
+  ignoreFocus?: boolean
 }
 const LeftToRightList: React.FC<LeftToRightListProps> = ({
```

Comment:
Not a change is this PR but File name doesn't match default export. Personally I don't mind too much but are conventions seem to be to have the default import reflect the file name.

As a side note, I have no idea what a `LeftRightList` is. Should we discuss some better names or do you think this is best?

Reply:
LeftToRightList is a key navigation div where you circle through focusable items via left and right arrow keys
https://gyazo.com/045c25c8d6250aaea50030143427ffe4

I agree naming is not ideal but I have honestly no idea what would be a good name without making it super long
I fied the name file